src/components/Guilds/common/Button/Button.test.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render, fireEvent, screen } from '@testing-library/react';
+import { Button } from './Button';
+import { themedComponent } from '../../../../utils/tests';
+
+test('renders button', async () => {
+  render(themedComponent(<Button>hola</Button>));
+  fireEvent.click(screen.getByText('hola'));
+});

```

Comment:
I don't see why we should lose the `.tsx` extension on this file.

Reply:
sure